### PR TITLE
Milestone 2: Portfolio Mobile Version

### PR DESCRIPTION
**Pull request by Saadat123456**
Complete Milestone 2 of Microverse Portfolio. Add Recent Works Section and About Me Section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,5 +59,129 @@ <h1>Hey there. I'm Saadat
             </ul>
             <a href="#" class="down-btn"><img src="img/Union.svg" alt="Down Button"></a>
         </section>
+        <section class="Works">
+            <div class="section-heading">
+                <h1>My Recent Works</h1>
```
Comment (ahmetbozaci) [inline index.html:64]: - [ ] You have already one h1 tag on the page. The page should include only one [h1 tag](https://www.w3schools.com/tags/tag_hn.asp#:~:text=Note%3A%20Only%20use%20one%20%3Ch1%3E%20per%20page%20%2D%20this%20should%20represent%20the%20main%20heading/subject%20for%20the%20whole%20page.%20Also%2C%20do%20not%20skip%20heading%20levels%20%2D%20start%20with%20%3Ch1%3E%2C%20then%20use%20%3Ch2%3E%2C%20and%20so%20on.). Please change to another hx tag.
- [ ] Under my recent work text green line is exists. To stick to the design please add it.

YOUR DESIGN
image
___

### ORIGINAL
image

File: index.html [hunk 2]
```diff
@@ -60,5 +59,129 @@ <h1>Hey there. I'm Saadat
...
+                        <li>html</li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>                
+            </div>
+        </section>
+        <section class="About">
+            <div class="about-headline">
+                <div class="section-heading">
+                    <h1>About me</h1>
```
Comment (ahmetbozaci) [inline index.html:142]: - [ ] You have already one h1 tag on the page. The page should include only one [h1 tag](https://www.w3schools.com/tags/tag_hn.asp#:~:text=Note%3A%20Only%20use%20one%20%3Ch1%3E%20per%20page%20%2D%20this%20should%20represent%20the%20main%20heading/subject%20for%20the%20whole%20page.%20Also%2C%20do%20not%20skip%20heading%20levels%20%2D%20start%20with%20%3Ch1%3E%2C%20then%20use%20%3Ch2%3E%2C%20and%20so%20on.). Please change to another hx tag.

File: index.html [hunk 3]
```diff
@@ -60,5 +59,129 @@ <h1>Hey there. I'm Saadat
             </ul>
             <a href="#" class="down-btn"><img src="img/Union.svg" alt="Down Button"></a>
         </section>
+        <section class="Works">
```
Comment (ahmetbozaci) [inline index.html:62]: - [ ] I see in this section you use flex. In the end, this website will be responsive so kindly change it to `display: grid`. 
Check-in [here](https://www.geeksforgeeks.org/comparison-between-css-grid-css-flexbox/#:~:text=CSS%20Grids%20helps,the%20same%20time.) why you should use grid.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️

### Hello @Saadat123456  ✋🏼 
#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- You stick to the design.✔️
- You know how to use flexbox very well.✔️
- You use HTML semantic elements correctly.✔️

___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

almost
coding



___

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_


_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
___
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: index.html (hunk 3 above)
Comment (ahmetbozaci) [reply index.html:62]: @Saadat123456 Hello, it is correct my mistake. You already used the grid for the works section. I probably saw the headline class.
image

Comment (Saadat123456) [reply index.html:62]: Thanks for replying. That's Great! Now I can submit for review.

**Review by leolpaz — APPROVED** [review]
Hi @Saadat123456 , 

Your project is complete!
Congratulations! 🎉🎉 


Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Saadat123456** [comment]
Thanks @leolpaz .

**Pull request merged**